docs/hipify-clang.md

```diff
@@ -213,23 +213,99 @@ For example:
 ./hipify-clang square.cu --cuda-path=/usr/local/cuda-12.3 -I /usr/local/cuda-12.3/samples/common/inc
 ```
 
-`hipify-clang` arguments are given first, followed by a separator `'--'`, and then the arguments you'd pass to `clang` if you
-were compiling the input file. For example:
+`hipify-clang` arguments are given first, followed by a separator `'--'`, and then the arguments you'd pass to `clang` if you were compiling the input file. For example:
 
 ```bash
 ./hipify-clang cpp17.cu --cuda-path=/usr/local/cuda-12.3 -- -std=c++17
 ```
 
+hipify-clang also supports the hipification of multiple files that might be specified in a single command line with absolute or relative paths, for instance:
+
+```bash
+./hipify-clang cpp17.cu ../../square.cu /home/user/cuda/intro.cu --cuda-path=/usr/local/cuda-12.3 -- -std=c++17
+```
+
 The [Clang manual for compiling CUDA](https://llvm.org/docs/CompileCudaWithLLVM.html#compiling-cuda-code) may be useful.
 
+## <a name="hipify-clang-using-json-compilation-database"></a>hipify-clang: using JSON Compilation Database
+
 For some hipification automation (starting from clang 8.0.0), it is also possible to provide a [Compilation Database in JSON format](https://clang.llvm.org/docs/JSONCompilationDatabase.html) in the `compile_commands.json` file:
 
 ```bash
 -p <folder containing compile_commands.json> or
 -p=<folder containing compile_commands.json>
 ```
 
-The compilation database should be provided in the `compile_commands.json` file or generated by clang based on CMake; options separator `'--'` must not be used.
+The compilation database should be provided in the `compile_commands.json` file or generated by clang based on CMake; multiple source files can be specified as well; options separator `'--'` must not be used.

```

Comment:
Recommended changes,

Add the following sentence as a note, as it seems crucial. Also, what happens if users accidentally use it? What is the recommended separator? 

**Note**: options separator `'--'` must not be used.

Reply:
What about adding the following:

Here is an example of the` compile_commands.json` usage: https://github.com/ROCm/HIPIFY/blob/amd-staging/tests/unit_tests/compilation_database/compile_commands.json.in

Reply:
Perfect for additional clarity. Examples always help. So, can we revise the original description to,

Only Clang options must be provided in compile_commands.json. hipify-clang options can be provided in the hipify-clang command line. You may refer to the following <example link> for more details. 

Note: Do not use the options separator '--'. A compilation error will occur if the hipify-only options are provided before the separator.

Reply:
Done